src/Pages/Verify/VerifyCenterDiv/VerifyCenterDiv.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react"
+import './VerifyCenterDiv.scss'
+import { useDispatch, useSelector } from "react-redux"
+
+import {PageTittle} from "../../../components/PageTittle"
+import { Input } from "../../../components/Input"
+import { Button } from "../../../components/Button"
+import { verify } from "../../../thunkAction/Verify";
+import { useNavigate } from "react-router-dom"
+
+
+export const VerifyCenterDiv = () => {
+    
+    const dataSignUp = useSelector((state: any) => state.verify);
+    const dispatch = useDispatch();
+    const navigate = useNavigate();
+
+    const [uid, setUid] = useState('')
+    const [token, setToken] = useState('')
+
+    const onUidChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setUid(event.target.value)
+    } 
+    const onTokenChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setToken(event.target.value)
+    } 
+
+    useEffect(() => {
+        if (dataSignUp.dataVerify == 'ok') {
+            navigate('/')
+        }
+    }, [dataSignUp])
+
+    const  verifyForm = (event: any) => {
```

Comment:
Ты не верифицируешь форму, а отправляешь. Как альтернатива `verifyUser`
```suggestion
    const  submitForm = (event: any) => {
```